plugins/gatsby-source-newrelic-sdk/src/navigationApi.js

```diff
@@ -155,7 +155,7 @@ const location = navigation.getOpenLauncherLocation(launcher);
           label: 'Example 2',
           sourceCode: `
 // You can also specify a nerdlet and stacked nerdlets to be opened.
-// You can combine artifacts from different nerdpacks.
+// You can combine nerdpack items from different nerdpacks.
```

Comment:
Even though these are in a js file, you can leave them because "artifacts" in this context is what we want to change.